packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -496,21 +497,26 @@ export class MenusContributionPointHandler {
             return Disposable.NULL;
         }
 
-        const selectedResource = () => {
+        const getSelectedResources = () => {
             const selection = this.selectionService.selection;
+            const resourceParams = [];
             if (TreeWidgetSelection.is(selection) && selection.source instanceof TreeViewWidget && selection[0]) {
-                return selection.source.toTreeViewSelection(selection[0]);
+                resourceParams[0] = selection.source.toTreeViewSelection(selection[0]);
             }
             const uri = this.resourceContextKey.get();
-            return uri ? uri['codeUri'] : undefined;
+            resourceParams[0] = uri ? uri['codeUri'] : undefined;
```

Comment:
This looks like it __always__ overrides the value at index `0`. Perhaps this should only run inside of an else branch?